shared/gemfiles/20220706/Gemfile

```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
```

Comment:
There's no need to have multiple native extension versions since it'll be too much of a hassle to maintain. Better merge all these directories into one.